spec/models/invenio_rdm_record_converter_spec.rb

```diff
@@ -42,17 +45,32 @@
           "affiliations": []
         }],
         "title": "#{generic_file.title.first}",
-        "additional_titles": [{
-          "title": "Secondary Title",
-          "type": "alternative_title",
-          "lang": "eng"
-        },
-        {
-          "title": "Tertiary Title",
-          "type": "alternative_title",
-          "lang": "eng"
-        }],
-        "formats": "application/pdf"
+        "additional_titles": [
```

Comment:
I changed this formatting a little, but I can change it back. I keep flip flopping on this... JSON always looks kind of nasty to me for whatever reason.